singleheader/amalgamate.py

```diff
@@ -114,27 +99,25 @@ def dofile(fid: str, prepath: str, filename: str) -> None:
 README = os.path.join(AMALGAMATE_OUTPUT_PATH, 'README.md')
 
 print(f'Creating {AMAL_H}')
-amal_h = open(AMAL_H, 'w')
+amal_h = open(AMAL_H, mode='w', encoding='utf8')
 print(f'/* auto-generated on {timestamp}. Do not edit! */', file=amal_h)
-for h in ALLCHEADERS:
+for h in ALL_C_HEADERS:
     doinclude(amal_h, h, f'ERROR {h} not found', h)
 
 amal_h.close()
-print()
-print()
 print(f'Creating {AMAL_C}')
-amal_c = open(AMAL_C, 'w')
+amal_c = open(AMAL_C, mode='w', encoding='utf8')
```

Comment:
@lemire unfortunately encoding='ascii' doesn't work

Reply:
utf8 is fine